### ISSUE TITLE
use of boot2docker (in runOpRp.sh) is deprecated

### ISSUE THREAD

**Issue by dtenenba** [rohe/pyoidc#266]
Docker on Mac no longer uses a virtual machine (boot2docker) so logic & commands should be the same as on Linux. Typically `HOST_IP` will be `127.0.0.1`. 

Also the `ifconfig` command in this shell script does not return anything on a macOS Sierra machine.

Also, I can't get the docker example to work, and the example docs are incorrect. They say that containers will be listening on ports 8088 and 8666 but `docker ps` tells me they are listening on 8092 and 8666. This makes me doubt whether `username@localhost:8093` is the correct UID to enter, since nothing is listening on port 8093. At any rate, all variations that I try result in a MaxRetriesExceeded error.

Since the docker example seems like the easiest point of entry to those who (like me) are new to OIDC, it would be great if it worked. Thanks.

**Comment by lwm** [comment]
I raised a similar issue with https://github.com/rohe/pyoidc/issues/254 and submitted https://github.com/rohe/pyoidc/pull/255 and https://github.com/rohe/pyoidc/pull/252 along the way.

I tried to remove a lot of the unmaintained stuff but it never got merged in https://github.com/rohe/pyoidc/pull/256.

If you have time to contribute, please think about digging in and trying to fix things :+1:

**Comment by dtenenba** [comment]
Thanks but the fact that there are no working examples makes it hard for me to know what to fix. And the fact that I'm a noob when it comes to OIDC. Plus it seems that PRs don't necessarily get merged?

I am struggling to figure out how to set up a working OP and RP, using Docker or not. Any help would be appreciated. Once I get things working maybe I'll be able to contribute back.

**Comment by lwm** [comment]
@dtenenba yes, I feel your pain. Similar sentiment here. I couldn't get anything working from the examples but did go on to use the library successfully once I grokked the source. Depends on how much time you have and what framework you're using with. Django has better documented alternatives ...

**Comment by rohe** [comment]
This is a feeble excuse but anyway. I used to have a group of 4 people working on pyoidc related things now it's just me. So some things have fallen by the wayside. The Docker support is definitely one of them. Documentation as you bpth have experienced another. So I have just taken up Luke on his offer to help out.
